src/axom/primal/CMakeLists.txt

```diff
@@ -64,8 +64,8 @@ set( primal_headers
 
 set( primal_sources
 
-     ## operators
-     operators/detail/intersect_impl.cpp
+     ## Empty source file
+     dummy.cpp
```

Comment:
To fix `lld: error: undefined hidden symbol` errors when linking HIP kernels to functions inside 
`intersect_impl.hpp`, I consolidated `intersect_impl.cpp` into the header file.

That left `primal` without any source files...

I tried making `primal` a header-only library as described in blt docs [here](https://llnl-blt.readthedocs.io/en/develop/api/target.html#blt-add-library), but that does not work for object libraries. So, I added an empty source file to satisfy the requirement.

@white238 I think this is not the ideal workaround. Any suggestions?

Reply:
@white238 would have the definitive answer here, but since he's currently working on removing object libraries from axom, the current solution seems reasonable to me.